packages/compass-shell/src/components/shell-header/shell-header.jsx

```diff
@@ -106,7 +105,23 @@ export const ShellHeader = ({
           onClick={onShellToggleClicked}
           aria-pressed={isExpanded}
         >
-          &gt;_MONGOSH
+          <GuideCue
```

Comment:
Now that we have this cue, would it make sense to remove blinking animation from this element?

Reply:
I didn't notice this cool animation until now!

Reply:
removed in [8f803c6](https://github.com/mongodb-js/compass/pull/4568/commits/8f803c6c3c54902343319d713a63d3186f51f04c)